src/shims/unix/solarish/foreign_items.rs

```diff
@@ -19,6 +18,11 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
     ) -> InterpResult<'tcx, EmulateItemResult> {
         let this = self.eval_context_mut();
         match link_name.as_str() {
+            "stack_getbounds" => {
```

Comment:
Please match the structure [used here](https://github.com/rust-lang/miri/blob/100c718ee7bf479df7763f78b4c268a05a109333/src/shims/unix/linux/foreign_items.rs#L205): use the same comment, put the shim at the end of the list, and guard it with `if this.frame_in_std()`